assets/sass/layouts/_responsive.scss

```diff
@@ -0,0 +1,27 @@
+@media (max-width: 1130px) {
+  .wrap {
+    padding: 0 15px;
+  }
+
+  #primary-nav {
+    width: 100%;
+    position: relative;
+    left: 0;
+    margin-top: -50px;
+  }
+
+  ul.menu {
+    li:hover ul {
```

Comment:
@nucro please follow BEM here as well. You will need to edit the class names in the markup and then use those class names in the SCSS